lib/associations/belongs-to-many.js

```diff
@@ -287,6 +287,16 @@ class BelongsToMany extends Association {
        */
       removeMultiple: 'remove' + plural,
       /**
+       * Un-associate all instances.
+       *
+       * @param {Object} [options] Options passed to `through.destroy`
```

Comment:
I should only modify the removeMultiple, and not create this destroyAssociation? If I do this, if someone use the removeMultiple and pass an empty array, the result would be different and will cause a lot of non expected results.
I just create another method, because the issue say that.